src/components/AlbumCard/AlbumCard.css

```diff
@@ -0,0 +1,104 @@
+:root {
+    --dark-color: #2A3439;
+    --primary-color: #36454F;
+    --secondary-color: #536878;
+    --third-color: #708090;
+    --forth-color: #91A3B0;
+    --fifth-color: #BEBFC5;
+    --light-color: #E5E4E2;
+    --lighter-color: #F5F5F5;
+    --silver-color: #C0C0C0;
+    
+}
```

Comment:
you don't need to define the variables here again, it is enough if you do that in the index.css. All other files can use it then.